doc/source/cluster/index.rst

```diff
@@ -3,227 +3,33 @@
-
-.. important:: Calling ``ray.init`` on your laptop will not work if using ``ray up``, since your laptop will not be the head node.
-
-Here is an example of using the Cluster Launcher on AWS:
-
-.. code-block:: shell
-
-    # First, run `pip install boto3` and `aws configure`
-    #
-    # Create or update the cluster. When the command finishes, it will print
-    # out the command that can be used to SSH into the cluster head node.
-    $ ray up ray/python/ray/autoscaler/aws/example-full.yaml
-
-You can monitor the Ray cluster status with ``ray monitor cluster.yaml`` and ssh into the head node with ``ray attach cluster.yaml``.
-
-.. _manual-cluster:
-
-Manual Ray Cluster Setup
+What is Distributed Ray?
 ------------------------
 
-The most preferable way to run a Ray cluster is via the :ref:`Ray Cluster Launcher <ref-automatic-cluster>`. However, it is also possible to start a Ray cluster by hand.
-
-This section assumes that you have a list of machines and that the nodes in the cluster can communicate with each other. It also assumes that Ray is installed
-on each machine. To install Ray, follow the `installation instructions`_.
-
-.. _`installation instructions`: http://docs.ray.io/en/master/installation.html
-
-Starting Ray on each machine
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-On the head node (just choose some node to be the head node), run the following.
-If the ``--port`` argument is omitted, Ray will choose port 6379, falling back to a
-random port.
-
-.. code-block:: bash
-
-  $ ray start --head --port=6379
-  ...
-  Next steps
-    To connect to this Ray runtime from another node, run
-      ray start --address='<ip address>:6379' --redis-password='<password>'
-
-  If connection fails, check your firewall settings and network configuration.
-
-The command will print out the address of the Redis server that was started
-(the local node IP address plus the port number you specified).
-
-**Then on each of the other nodes**, run the following. Make sure to replace
-``<address>`` with the value printed by the command on the head node (it
-should look something like ``123.45.67.89:6379``).
-
-Note that if your compute nodes are on their own subnetwork with Network
-Address Translation, to connect from a regular machine outside that subnetwork,
-the command printed by the head node will not work. You need to find the
-address that will reach the head node from the second machine. If the head node
-has a domain address like compute04.berkeley.edu, you can simply use that in
-place of an IP address and rely on the DNS.
-
-.. code-block:: bash
-
-  $ ray start --address=<address> --redis-password='<password>'
-  --------------------
-  Ray runtime started.
-  --------------------
-
-  To terminate the Ray runtime, run
-    ray stop
-
```

Comment:
Are there any instructions anywhere for starting Ray clusters manually? Where have the manual Ray start commands been moved to?

This is important to keep somewhere because there will still be a subset of users that cannot use the Ray cluster launcher but need to start a ray cluster.